pkg/contexts/ocm/accessmethods/npm/method.go

```diff
@@ -96,43 +96,70 @@ func (a *AccessSpec) GetInexpensiveContentVersionIdentity(access accspeccpi.Comp
 	return ""
 }
 
-// PackageUrl returns the URL of the NPM package (Registry/Package/Version).
+// PackageUrl returns the URL of the NPM package (Registry/Package).
 func (a *AccessSpec) PackageUrl() string {
+	return a.Registry + path.Join("/", a.Package)
+}
+
+// PackageVersionUrl returns the URL of the NPM package-version (Registry/Package/Version).
+func (a *AccessSpec) PackageVersionUrl() string {
 	return a.Registry + path.Join("/", a.Package, a.Version)
 }
```

Comment:
If a.Registry ends with a / the resulting URL will probably not work. We had this problem when testing the maven part.

Reply:
done

Reply:
I don't know whether :8 (https://) is a good idea. May be just check whether the Registry has a / as suffix and adding a leading suffix if this is not the case, only.

Package and version may not contain a /, anyway.

Reply:
looks fine